gdx/src/com/badlogic/gdx/math/DelaunayTriangulator.java

```diff
@@ -85,17 +85,15 @@ public ShortArray computeTriangles (float[] points, int offset, int count, boole
 			if (value > ymax) ymax = value;
 		}
 		float dx = xmax - xmin, dy = ymax - ymin;
-		float dmax = (dx > dy ? dx : dy) * 20f;
-		float xmid = (xmax + xmin) / 2f, ymid = (ymax + ymin) / 2f;
 
 		// Setup the super triangle, which contains all points.
 		float[] superTriangle = this.superTriangle;
-		superTriangle[0] = xmid - dmax;
-		superTriangle[1] = ymid - dmax;
-		superTriangle[2] = xmid;
-		superTriangle[3] = ymid + dmax;
-		superTriangle[4] = xmid + dmax;
-		superTriangle[5] = ymid - dmax;
+		superTriangle[0] = xmin - dx;
```

Comment:
I'm wondering, does choosing a smaller super triangle result in better precision?
If yes, I use this code in my triangulator:
```
Vertex v = new Vertex((minx - 1), (miny - height));
Vertex u = new Vertex((minx - 1), (miny + height * 2));
Vertex w = new Vertex((minx + width * 1.5 + 1), (miny + height * 0.5));
```
which results in an even smaller rectangle.
But not sure whether my implementation has some quirks, just throwing it in here.

Reply:
@Berstanio The truth is that I really don't know what's the impact of the supertriangle.
I just guessed that in terms of performance the closer you get to the points cloud the better. But it's just a guess.
In terms of rounding errors it seems it needs to be big enough to avoid them

Regarding your triangle, it's not always smaller. It depends on the size of the cloud. For clouds with sizes smaller than one I guess it will be bigger

In any case, with the last commit I included a new way of computing the supertriangle that I think it's better, as close a s possible to the cloud but leaving enough room to possibly avoid round errors

Reply:
> Regarding your triangle, it's not always smaller. It depends on the size of the cloud. For clouds with sizes smaller than one I guess it will be bigger

Interesting, good to know, thank you! I will revisit my code then. I'm using integer positions in my code, so maybe thats why it worked for me.

I will also look at your new supertriangle code!

Reply:
Okay, I thought more about the supertriangle stuff.

I'm not sure whether the already code handles this case, but I think this might be problematic:
Consider only  three points, that are close to being co-linear. They will have a huge circumcircle, so that they might contain vertices of the super-triangle. This _might_ break the triangulator code, as triangles that shouldn't violate the delaunay condition might break it now.
The original issue also mentions near co-linear points, so it might be related.

If this is an issue, the proper way to fix it might be to just move the supertriangle vertices to infinity. And for the circumcircle check we could use the half-space then.

But again, I don't really understand the code very well and it might not be an issue here.

Reply:
@jose-puente 
I think my suspicion was correct. Take this test:
```
	public static class NearlyColinearTest extends DelaunayTriangulatorTest {
		@Parameters
		public static Object[] nearlyColinear () {
			float[] hs = new float[6];
			hs[0] = 1;
			hs[1] = 1;

			hs[2] = 1;
			hs[3] = 100;

			hs[4] = 10f;
			hs[5] = 200;
			return new Object[]{hs};
		}

		@Override
		protected int expectedTrianglesCount () {
			return 1;
		}
	}
```
This fails with your current super triangle, but passes with the old one. 
I think we should really move the super triangle to infinity.

Reply:
Cool, we should add the `nearlyColinear` test(s). It's great we have tests!

It'd probably be fine to make the super triangles 10% larger than the vertices' AABB. It should be large enough to not make nearly colinear triangles, but not so large that it reduces precision.